lms/templates/fields/field_social_link_account.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="u-field-value field <%- linkClass %>">
+    <span class="u-field-title field-label"><%- title %></span>
+    <span class="u-field-social-help"><%- subTitle %></span>
+    <a class="u-field-link" id="u-field-link-<%- id %>" href="<%- linkHref %>" aria-describedby="u-field-message-help-<%- id %>">
+        <span class="sr"><%- screenReaderTitle %></span>
```

Comment:
Why is the screen reader title different than the text value?  This creates a condition where the dev can forget to keep these values in sync, which is exactly whats going on here.  The visual label says "Unlink this account" but the screen reader hears "Connect your google account".  These are the worst types of bugs because they can only be discovered by using a screen reader, and they cannot see the screen to understand what the bug is.

![screenshot of dom that shows out of sync values](https://cloud.githubusercontent.com/assets/444368/15323224/039be4b4-1c0f-11e6-94d7-d0fe17d56523.png)